src/components/ContainerSettingsAdvanced.react.js

```diff
@@ -60,6 +68,7 @@ var ContainerSettingsAdvanced = React.createClass({
             <p><label><input type="checkbox" checked={this.state.tty} onChange={this.handleChangeTty}/>Allocate a TTY for this container</label></p>
             <p><label><input type="checkbox" checked={this.state.openStdin} onChange={this.handleChangeOpenStdin}/>Keep STDIN open even if not attached</label></p>
             <p><label><input type="checkbox" checked={this.state.privileged} onChange={this.handleChangePrivileged}/>Privileged mode</label></p>
+            <p><label><input type="checkbox" checked={this.state.restartPolicy} onChange={this.handleChangeRestartPolicy}/>Start on docker startup</label></p>
```

Comment:
I would perhaps label this differently, something like `Enable 'always' restart policy`
By definition, the `always` policy does a lot more than just start when docker starts:
```
Always restart the container regardless of the exit status. When you specify always, the Docker daemon will try to restart the container indefinitely. The container will also always start on daemon startup, regardless of the current state of the container.
```